anvil/config.py

```diff
@@ -9,12 +9,35 @@
 from reportengine.configparser import ConfigError, element_of, explicit_node
 
 from anvil.core import TrainingOutput
+from anvil.train import (
+    adam,
+    adadelta,
+    stochastic_gradient_descent,
+    reduce_lr_on_plateau,
+)
 from anvil.models import RealNVP
 from anvil.geometry import Geometry2D
-from anvil.distributions import normal_distribution, phi_four_action
+from anvil.distributions import *
 
 log = logging.getLogger(__name__)
 
+BASE_OPTIONS = {
```

Comment:
so now I'm wondering if we should have base options defined distributions and then just import the dictionary